docs/network-flow-visibility.md

```diff
@@ -43,6 +44,45 @@ For visualizing the network flows, Antrea monitors the flows in Linux conntrack
 module. These flows are converted to flow records, and then flow records are post-processed
 before they are sent to the configured external flow collector.
 
+## Getting Started
+
+To get started, let's ensure you have installed Antrea with the Flow Exporter
+feature enabled by setting the following config parameter in `antrea.yaml`. 
+
+```yaml
+  antrea-agent.conf: |
+    # FeatureGates is a map of feature names to bools that enable or disable experimental features.
+    featureGates:
+    # Enable flowexporter which exports polled conntrack connections as IPFIX flow records from each
+    # agent to a configured collector.
+      FlowExporter: true
+```
+
+To enable both the Grafana Flow Collector and the
+[NetworkPolicy Recommendation](./networkpolicy-recommendation.md) features in
```

Comment:
Maybe remove "features in Theia"?

"both Grafana Flow Collector and NetworkPolicy Recommendation"